entities/regions.py

```diff
@@ -0,0 +1,14 @@
+from random import randint
+from random import choice
+from config import Config
+
+class Regions(object):
+    region = []
+    def create_region(self):
+        for i in range(Config.number_of_regions):
+            self.region_id = i + 1
+            for j in range(Config.number_of_subregions):
+                self.hub = [int((j + 1 + i * 4) * Config.subregion_length - (Config.subregion_length / 2)),
+                            int(Config.subregion_length / 2)]
+                self.region.append([[(j + i * 4) * Config.subregion_length, (j + 1 + i * 4) * Config.subregion_length,
```

Comment:
Don't append a list (why are you so fond of lists?). Use a Region object, so you may obtain location, id, and hubs directly from the object (as e.g. `regions[1].hub` instead of `regions[1][2]`)

Reply:
It's not that I am particularly fond of lists, but I am still trying to get the full grasp of objects and classes. Lists are my familiar alternative.

I've dug into the theory behind classes and objects again and will try to re-arrange it.